components/html-block/html-block.js

```diff
@@ -257,6 +261,10 @@ class HtmlBlock extends RtlMixin(LitElement) {
 
 	_contextChanged() {
 		if (!this._contextObserverController) return false;
+		if (!this._contextKeys) {
```

Comment:
Given the asynchronous nature of the controller instantiation now, it's possible that the controller won't yet exist when `firstUpdated` runs and we can't guarantee this will be initialized yet.